test/api/requests/site.test.js

```diff
@@ -115,6 +116,45 @@ describe('Site API', () => {
         })
         .catch(done);
     });
+
```

Comment:
These tests seem reasonable. I would prefer them to be written in the newer style, there are examples to which I could point, but it's not a big deal to leave them as is.

Reply:
I mean, I almost just added my single expectation
```
expect(responseSite).to.include.keys('canEditLiveUrl','canEditDemoUrl');
```
to the `'should not render any sites not associated with the user'` test above, but decided to copy that test wholesale to make this new concern more explicit. 

If the tests should be rewritten in the newer style then they should be, but I vote that that's outside the scope of this PR, and for the moment it would be weird for this one new test to be in a different style than the one it's derived from. Maybe we should create an issue to update the test style? Or maybe we have one?